### ISSUE TITLE
Don't generate model at all (but instead refer back to the original classes)

### ISSUE THREAD

**Issue by muhmud-v** [swagger-api/swagger-codegen#3048]
##### Description

I've searched around, generally & in the issues, but can't find a way to do this. I'd like no model at all to be generated, but instead have the generated code refer back to the original model classes. The --import-mapping option is just too cumbersome. I have a separate jar for my models and I'd just like to use that, even if I have to fix my pom file manually.

##### Swagger-codegen version

2.6.1

##### Suggest a Fix

Have an option that just doesn't generate models at all, but assumes that the original model classes will be on the class path.

**Comment by fehguy** [comment]
How will the operations know what to import?  Regardless you can generate just operations and supporting files. But the import mapping option is really the smartest way to do this, since it does imports in all operations. 

If it's cumbersome, please provide "what" is cumbersome about it.

**Comment by muhmud-v** [comment]
Yeah, just figured out that I can do just apis and supporting files, which with the modelPackage parameter provides output a lot closer to what I want. I still need to do some manual edits, as my models are in grouped into separate sub-packages.

Thinking about it, I see what you mean; the codegen is using swagger.json, not reflection, so doesn't know anything about the original classes, hence wouldn't know what to import. 

It's cumbersome because I have a fair number of models, and I don't want to have to list all of them & then maintain that list going forward. Some kind of wildcard mapping would be really useful.

**Comment by fehguy** [comment]
Sure, a wildcard mapping is possible--think of what the syntax would be and it can be added

**Comment by muhmud-v** [comment]
This might be slightly biased to my use case, but would be great to be able to say all models used in a particular api path prefix, with * to skip parts of the path, should have it's models imported from a particular package, e.g.

`--imports /api/customer=com.company.models.customer,/api/customer/*/order=com.company.models.order`

Also, as an aside, when everything was said & done, I had to add `register(JacksonJsonProvider.class)` to `buildHttpClient()` in the generated `ApiClient` class to get this to work without getting a "Response reader not found" exception. I don't think this is because of something I messed with, but's its possible.

Something else that might need to go in a different swagger project, but why doesn't swagger.json have a schema attribute for the response of a POST/PUT? I've specified the Response type in my annotations, but it doesn't seem to get used. The return type of the methods in the generated client then become void.

**Comment by fehguy** [comment]
ok skipping in the middle of the path isn't really possible but a trailing wildcard could work as a generic import.

**Comment by muhmud-v** [comment]
If the path in the middle can't skipped, would it still be able to contain a parameter reference, i.e. /api/customer/{customerID}/order? That would be fine as well. Would definitely want the wildcard on the end.

**Comment by fehguy** [comment]
I mean, middle of the "package" not the "http path".

**Comment by muhmud-v** [comment]
So, I would always need to fully specify package names? That's what I was expecting, if I understand you correctly. So, would my example above work?
